cmd/openscap-plugin/xccdf/datastream.go

```diff
@@ -160,6 +161,38 @@ func populateProfileVariables(dsProfile *xmlquery.Node, parsedProfile *xccdf.Pro
 	return parsedProfile, nil
 }
 
+func populateProfileRules(dsProfile *xmlquery.Node, parsedProfile *xccdf.ProfileElement) (*xccdf.ProfileElement, error) {
+	if parsedProfile.Selections == nil {
+		parsedProfile.Selections = []xccdf.SelectElement{}
+	}
+
+	profileRules, err := getDsElements(dsProfile, "xccdf-1.2:select")
+	if err != nil {
+		return parsedProfile, fmt.Errorf("error finding 'select' elements in profile: %s", err)
```

Comment:
```suggestion
		return parsedProfile, fmt.Errorf("error finding 'select' elements in profile: %w", err)
```
For existing errors, you can use %w instead of %s to preserve all of the error information.